plugins/configuration.rb

```diff
@@ -0,0 +1,92 @@
+module Jekyll
+  class ConfigurationBlock < Liquid::Block
+    TYPE_LINKS = {
+      'action'       => '/docs/automation/action/',
```

Comment:
This should link to the script syntax page https://home-assistant.io/docs/scripts/